ftl/common/builder.py

```diff
@@ -123,8 +123,12 @@ def __init__(self, ctx, namespace, args, cache_version_str,
         self._target_creds = docker_creds.DefaultKeychain.Resolve(
             self._target_image)
         self._transport = transport_pool.Http(httplib2.Http, size=_THREADS)
-        self._base_image = docker_image.FromRegistry(
-            self._base_name, self._base_creds, self._transport)
+        if args.tar_base_image_path:
```

Comment:
might want to put this in a try/except to make sure the tarball is parsed to an image correctly.

Reply:
good idea, adding